### PR TITLE
Fixed the default API port

### PR DESCRIPTION
**Pull request by andrasfuchs**
Fixed the default API port for the Stratis daemon

### CODE REVIEW

File: src/Stratis.Bitcoin/Configuration/NodeSettings.cs [hunk 1]
```diff
@@ -215,7 +215,7 @@ public static NodeSettings Default(Network network = null, ProtocolVersion proto
 
             nodeSettings.RequireStandard = config.GetOrDefault("acceptnonstdtxn", !(nodeSettings.RegTest || nodeSettings.Testnet));
             nodeSettings.MaxTipAge = config.GetOrDefault("maxtipage", DefaultMaxTipAge);
-            nodeSettings.ApiUri = config.GetOrDefault("apiuri", new Uri("http://localhost:37220"));
+            nodeSettings.ApiUri = config.GetOrDefault("apiuri", (nodeSettings.Network == Network.StratisMain || nodeSettings.Network == Network.StratisTest || nodeSettings.Network == Network.StratisRegTest) ? new Uri("http://localhost:37221") : new Uri("http://localhost:37220"));
```
Comment (bokobza) [inline src/Stratis.Bitcoin/Configuration/NodeSettings.cs]: indeed, good spot.
Can you please move the endpoints to be constants? look in StratisBitcoinFullNode\src\Stratis.BreezeD\Program.cs

**Comment by bokobza** [comment]
@andrasfuchs can you please resolve the conflict?

File: src/Stratis.Bitcoin/Configuration/NodeSettings.cs (hunk 1 above)
Comment (dangershony) [reply src/Stratis.Bitcoin/Configuration/NodeSettings.cs]: While we have a more wider plan to refactor Network for now I would actually say we add a new property to Network and call it ApiEndpoint and set it at network level.

Comment (bokobza) [reply src/Stratis.Bitcoin/Configuration/NodeSettings.cs]: @andrasfuchs I suggest you leave it for now, I'm working on a task that's moving all of these constants and default values around.

Comment (andrasfuchs) [reply src/Stratis.Bitcoin/Configuration/NodeSettings.cs]: Alright, cool! I'll keep my eyes open for those changes then :)

**Comment by bokobza** [comment]
Great, thank you!

**Comment by bokobza** [comment]
@andrasfuchs  sorry I had to revert this as the build was failing.
Can you please fix and resubmit this? also please make sure that the unit tests are passing.

**Comment by andrasfuchs** [comment]
No worries, I'll redo both of my pull requests tomorrow and I'll be more thorough with the merging and tests.

**Pull request merged**